app/models/direct_file_data.rb

```diff
@@ -173,72 +135,40 @@ def surviving_spouse=(value)
     end
   end
 
-  def spouse_date_of_death
-    df_xml_value(__method__)
-  end
-
   def spouse_date_of_death=(value)
     if value.present?
       create_or_destroy_df_xml_node(__method__, value, after="IndividualReturnFilingStatusCd")
       write_df_xml_value(__method__, value)
     end
   end
 
-  def spouse_name
-    df_xml_value(__method__)
-  end
-
   def spouse_name=(value)
     if value.present?
       write_df_xml_value(__method__, value)
     end
   end
 
-  def mailing_city
-    df_xml_value(__method__)
-  end
-
   def mailing_city=(value)
     write_df_xml_value(__method__, value)
   end
 
-  def mailing_street
-    df_xml_value(__method__)
-  end
-
   def mailing_street=(value)
     write_df_xml_value(__method__, value)
   end
 
-  def mailing_apartment
-    df_xml_value(__method__)
-  end
-
   def mailing_apartment=(value)
     write_df_xml_value(__method__, value)
   end
 
-  def mailing_state
-    df_xml_value(__method__)
-  end
-
   def mailing_state=(value)
     write_df_xml_value(__method__, value)
   end
 
-  def mailing_zip
-    df_xml_value(__method__)
-  end
-
   def mailing_zip=(value)
     write_df_xml_value(__method__, value)
   end
 
-  def fed_tax
-    df_xml_value(__method__)&.to_i || 0
-  end
-
-  def fed_tax=(value)
+  def fed_tax_amt=(value)
```

Comment:
how come `fed_tax_amt` from DirectFileData can be set? Most of the attributes are read-only here, so just trying to understand why some of them have setters/writers and some don't.

Reply:
so the setters are for this page that can only be accessed by our team in demo environments: right after the df data transfer there's a page where you can modify some of the data i.e. for acceptance testing purposes. i think which ones are editable is a liiiitle random (some we preemptively made setters for, others were requested by product, etc)